spec/paths/v1@enterprise@{enterprise_id}@application@{application_id}@.yaml

```diff
@@ -0,0 +1,70 @@
+get:
```

Comment:
Should we be making changes to old API also. Can we set deprecation notice for this. @tr0n-p46

Reply:
What should we do about this?